patch/atmega128a.yaml

```diff
@@ -0,0 +1,53 @@
+_svd: ../svd/atmega128a.svd
+
+_include:
+  - "common/ac.yaml"
+  - "common/adc.yaml"
+  - "common/spi.yaml"
+  - "common/usart.yaml"
+  - "timer/atmega128a.yaml"
+
+USART?:
+  UCSR?C:
+    _modify:
+      UMSEL?:
+        bitRange: "[7:6]"
+
+# Patches for the Two-Wire-Interface peripheral (i2c)
+#
+# - Make the Control Register read-write with only the
+#   "Write Collision Flag" being read-only
+# - Fix the Prescaler enumerated values
+TWI*:
+  TWCR:
+    _modify:
+      TWWC:
+        access: read-only
+  TWSR:
+    _modify:
+      TWS:
+        access: read-only
+    TWPS:
+      _replace_enum:
+        PRESCALER_1: [0, "Prescaler Value 1"]
+        PRESCALER_4: [1, "Prescaler Value 4"]
+        PRESCALER_16: [2, "Prescaler Value 16"]
+        PRESCALER_64: [3, "Prescaler Value 64"]
```

Comment:
You had mentioned some problems with USART and TWI.  I suppose these lines here are your custom patches for those peripherals to fix the problems you talked about?

Reply:
Yes, these lines fixed the errors mentioned here: https://github.com/DaanDekoningKrekels/avr-device/commit/cf4ee34701f95449c5954c9bc3d710e14bbac146#commitcomment-101614870 . 
Right now, it does not give any errors, but I am not yet 100% sure about the correctness...

Reply:
Ah, didn't see that.  Changes look good to me.  I'll merge this as-is and we can always go back and fix more things later on.